app/grandchallenge/reader_studies/templatetags/get_ground_truth.py

```diff
@@ -0,0 +1,10 @@
+from django import template
+
+register = template.Library()
+
+
+@register.simple_tag
+def get_ground_truth(obj, image, question):
+    """Get the auth token for the user."""
```

Comment:
Update the docstring

Reply:
Changed it locally. I'll include it in the next PR (making a separate PR for would be a bit much I'd say).